jba/src/inspections/analysis.py

```diff
@@ -48,23 +65,33 @@ def find_not_fixed_unique_inspections(group: pd.DataFrame) -> Set[str]:
 
 def get_unique_inspections_stats(
     df: pd.DataFrame,
+    file: Optional[str],
     inspections_to_ignore: List[str],
     normalize: bool = True,
 ) -> pd.DataFrame:
     unique_inspections = (
-        df.groupby(SubmissionColumns.GROUP.value).apply(find_unique_inspections).explode().value_counts()
+        df.groupby(SubmissionColumns.GROUP.value)
+        .apply(lambda group: find_unique_inspections(group, file))
+        .explode()
+        .value_counts()
```

Comment:
The same comment. I know it is a prototype, but please avoid this amount of the same code